panoptes_aggregation/reducers/user_skill_reducer.py

```diff
@@ -114,10 +116,10 @@ class instead.
         relevant_reduction : dict
             Dictionary containing the `subject_difficulty` array that gives
             the difficulty of the all the subjects seen by the user
-        binary : bool
+        mode : str
             Whether to run the reducer in binary (True/False) mode or k-class mode
```

Comment:
Update doc string.

Reply:
Yes -- sorry! We had planned to add documentation for all our changes as a second pass after testing the framework, so these updates got left out. Fixing it now...